server/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "gpay-group-buy-server",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:googleinterns/gpay-group-buy.git",
+  "author": "Karen Frilya Celine <karenfrilya@google.com>",
+  "license": "MIT",
```

Comment:
I think we are using "Apache-2.0" not "MIT" license!

Reply:
Right! Thanks for noticing :)